data/test/news/c0000000005o.json

```diff
@@ -159,6 +159,78 @@
             ]
           }
         },
+        {
```

Comment:
Thanks for updating a Test article to capture the two different cases - an image with caption and without a caption, but both having the copyright text.  
Great that you've also updated the Readme file as well. 💯